packages/@snowpack/plugin-webpack/plugin.js

```diff
@@ -33,9 +33,17 @@ module.exports = function plugin(config, args) {
     throw new Error("Output Pattern for CSS must end in .css");
   }
 
+  // Webpack handles minification, so its safe to disable 
+  // Snowpack's default minifier.
+  config.buildOptions.minify = false;
+  // Webpack needs the build directory to be cleaned so that 
+  // it doesn't bundle old builds.
+  config.buildOptions.clean = true;
```

Comment:
webpack actually doesn’t erase the build directory, but granted, it does usually build into a different location from source (whereas here source == dest). I’m on the fence about this, because there have been times where I’ve needed to place things in a webpack build from outside webpack so I wouldn’t expect this directory to be cleared.

Reply:
I guess I’m saying I think webpack devs expect the build location to not be cleared each time, and usually have a way of handling this.

Reply:
Yea, I'll expand on this in the comment. Since webpack uses non-deterministic file hashing, the second time you run a build you end up with outdated files in your build directory. I did some quick looking, and more build tools clear the build directory by default than you'd think (because of this issue.)

Reply:
Oh right which is a problem because we’re re-reading them. Normally webpack just ignores it because it never reads from all the old builds.

What if, instead, we had webpack just ignore reading files that are hashed? I think the webpack bundler only outputs a few JS files: `build/index-[hash].js`, `build/lib-[hash].js`, etc.

If that’s a pain I‘m in favor of clearing out the build directory. I was just anticipating someone asking us to not clear the build directory with webpack

Reply:
Bundled dev environments have the same issue though, right? 
- I build, `build/` contains new unique files
- I build again, `build/` now contains both old and new unique files
- etc. etc.